Ports/ncurses/Ncurses.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+PORT_DIR=Ncurses
+INSTALLOPTS="DESTDIR=$SERENITY_ROOT/Root/"
+
+function fetch() {
+    (
```

Comment:
Don't use a subshell, and instead of git apply, use:

    run_patch fix-ncurses.patch -p1

(it'll save you from having to find the right directory)

Also, even if it's obvious, do add a comment explaining what the patch is for, for future-us reference -- see the bash port for an example :). I think we should try enforce this as much as possible, as I've been in a few distro maintenance roles in the past, and figuring out what random patches do 2-3 years after they were written by someone else is *awful*.

Reply:
Ouch yeah. I'll rename the patch to something useful. Also, `run_patch` looks magic! (: